test/common/alerts_exist.go

```diff
@@ -0,0 +1,281 @@
+package common
+
+import (
+	"bytes"
+	goctx "context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	corev1 "k8s.io/api/core/v1"
+
+	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+var expectedRules = map[string][]string{
+	"/redhat-rhmi-middleware-monitoring": {
```

Comment:
Can we sync up on the structure of these rules? I'd like to understand this structure some more, mainly out of curiosity.

Reply:
Sure